charts/cryostat/README.md

```diff
@@ -105,12 +106,26 @@ A Helm chart for deploying [Cryostat](https://cryostat.io/) on Kubernetes and Op
+| `openshiftOauthProxy.tokenAccessReview.group`            | The OpenShift resource group that the TokenAccessReview will be performed for. See https://github.com/openshift/oauth-proxy/?tab=readme-ov-file#delegate-authentication-and-authorization-to-openshift-for-infrastructure                                                                                                   | `""`                                   |
+| `openshiftOauthProxy.tokenAccessReview.resource`         | The OpenShift resource that the TokenAccessReview will be performed for.                                                                                                                                                                                                                                                    | `pods`                                 |
+| `openshiftOauthProxy.tokenAccessReview.subresource`      | The OpenShift resource that the TokenAccessReview will be performed for.                                                                                                                                                                                                                                                    | `exec`                                 |
+| `openshiftOauthProxy.tokenAccessReview.name`             | The OpenShift resource name that the TokenAccessReview will be performed for.                                                                                                                                                                                                                                               | `""`                                   |
+| `openshiftOauthProxy.tokenAccessReview.namespace`        | The OpenShift namespace that the TokenAccessReview will be performed for.                                                                                                                                                                                                                                                   | `{{ .Release.Namespace }}`             |
+| `openshiftOauthProxy.tokenAccessReview.verb`             | The OpenShift resource name that the TokenAccessReview will be performed for.                                                                                                                                                                                                                                               | `create`                               |
+| `openshiftOauthProxy.tokenAccessReview.version`          | The OpenShift resource version that the TokenAccessReview will be performed for.                                                                                                                                                                                                                                            | `""`                                   |
```

Comment:
Should we combine the tokenAccessReview into subjectAccessReview to match the operator's implementation?
https://github.com/cryostatio/cryostat-operator/blob/b300689a645e3741ccd5468e09acf20b5b9fb4d2/api/v1beta2/cryostat_types.go#L484-L496